tachyon/math/polynomials/univariate/radix2_evaluation_domain.h

```diff
@@ -223,6 +232,43 @@ class Radix2EvaluationDomain : public UnivariateEvaluationDomain<F, MaxDegree>,
     OutInHelper(evals, start_gap);
   }
 
+  CONSTEXPR_IF_NOT_OPENMP void DegreeAwareFFTInPlace(
+      std::vector<Evals>& evalss) const {
```

Comment:
```suggestion
      std::vector<Evals>& evals) const {
```

Reply:
I was also not so sure about what to use here. `evals` is used for single instance of `Evals` across the repository. Do you think it's fine to use `evals` for vector and `eval` for single instance here?

Reply:
Hmm but technically evals is correct for the single instance so let's just use evalss for vector of evals.